packages/dashboard-frontend/src/services/bootstrap/PreloadData.ts

```diff
@@ -128,8 +129,16 @@ export class PreloadData {
       this.watchNamespaces(defaultNamespace);
     }
 
-    const { requestDwDevfiles } = DwPlugins.actionCreators;
-    await requestDwDevfiles(`${settings.cheWorkspacePluginRegistryUrl}/plugins/${settings['che.factory.default_editor']}/devfile.yaml`)(this.store.dispatch, this.store.getState, undefined);
+    const { requestDwDefaultEditor } = DwPluginsStore.actionCreators;
+    try {
+      await requestDwDefaultEditor(settings)(this.store.dispatch, this.store.getState, undefined);
+    } catch (e) {
+      const message = `Not able to create workspace due required resources failed to load: ${e}`;
```

Comment:
I wonder if it makes sense to change this to be something like:
```suggestion
      const message = `Required sources failed when trying to create the workspace: ${e}`;
```
or
```suggestion
      const message = `Unable to create the workspace. Requesting default editor failed: ${e}`;
```

Reply:
Nice suggestions, thanks!